### PR TITLE
Update to KF Conformance RFC after initial PR.

### PR DESCRIPTION
**Pull request by james-jwu**
PR #524 got merged before I could make the following changes. Main changes:
- Add more explicit "entitlement" section for distribution and application
- Used "Kubeflow Native" for application

### CODE REVIEW

**Comment by jbottum** [comment]
@james-jwu thanks for bring this to forward.   Let's complete this draft and create a version 1.

**Comment by thesuperzapper** [comment]
@jbottum ???

Why was this merged, there was still active discussion?

**Comment by thesuperzapper** [comment]
@james-jwu can you follow up with a separate PR that addresses the typos I raised?

**Comment by ca-scribner** [comment]
@jbottum why did you ask for feedback then?  There's ~10 comments that are note addressed, either by an update to the doc or an "I see what you're saying, but think we should not do that".  At the community meeting you asked for feedback on that document last week, and there's been some lively discussion on it since.  Merging arbitrarily like this just turns that effort into a waste of time

**Comment by thesuperzapper** [comment]
@jbottum I think you might have mistaken this PR for the previously merged "version 1" of this proposal https://github.com/kubeflow/community/pull/524

That's fine, it just means @james-jwu (or someone else) need to raise a new PR to address the typos raised in my comments above.

**Comment by james-jwu** [comment]
I am working on an update.

File: proposals/kubeflow-conformance-program-proposal.md [hunk 1]
```diff
@@ -74,9 +81,15 @@ We expect this test to evolve, due to the ambiguity of “Kubenetes Application
...
+The first version of the test verifies that the application is integrated with Kubeflow Pipelines. Pipeline and Metadata are the binding “glue” for other Kubeflow components. Metadata generation is automatic when Kubeflow Application conforms to standard Kubeflow Pipelines component interface.
 
-Kubeflow Application is entitled to refer to the application as "Kubeflow Native" or <TBD>. The application may be listed under an application catalog (to be created) under Kubeflow project. The naming of the application still needs to follow [Kubeflow Brand Guidelines](https://github.com/kubeflow/community/blob/master/KUBEFLOW_BRAND_GUIDELINES.pdf).
+**Entitlement**
+- Conformant applications are entitled to refer to the application as “Kubeflow Native” or <TBD>. Example usage:
+   - Appending “(Kubeflow Native)” to the distribution name: e.g. SUPER-TRAINER (Kubeflow Native)
+   - Reference the Kubeflow Native designation in discussion with customers, or on public documentation
+   - Display a logo (to be designed) on the public website and documentation of the application
+- The application may be listed under an application catalog (to be created) under Kubeflow project.
```
[the remark this answers is not in the document]
Comment (james-jwu) [reply proposals/kubeflow-conformance-program-proposal.md:91]: I am thinking about a web page like https://www.cncf.io/certification/software-conformance/, hosted on https://www.kubeflow.org/.

File: proposals/kubeflow-conformance-program-proposal.md [hunk 2]
```diff
@@ -60,7 +67,7 @@ The first version of conformance aims to be inclusive of current components in K
 
 # 
 
-# Kubeflow Application
+# Kubeflow Native (for Kubeflow Application)
```
[the remark this answers is not in the document]
Comment (james-jwu) [reply proposals/kubeflow-conformance-program-proposal.md:70]: Fixed in #528

File: proposals/kubeflow-conformance-program-proposal.md [hunk 3]
```diff
@@ -17,15 +17,22 @@ This document aims to define conformance criteria for the following usage of Kub
...
+# Certified Kubeflow (for Kubeflow Distribution)
 
 **A conformant Kubeflow Distribution is certified to provide a set of core functionalities and API integration options.**
 
 The tests will be designed in a way similar to [Kubernetes conformance program](https://github.com/cncf/k8s-conformance).  
 
 The tests will be versioned. Each versioned certification is valid for 1 year. After 1 year, recertification against the latest version of the test will be required to maintain certification standing.
 
-Conformant distributions is entitled to refer to the distribution as "Certified Kubeflow". The distribution can be listed under a partner page under the Kubeflow project. The naming of the distribution still needs to follow [Kubeflow Brand Guidelines](https://github.com/kubeflow/community/blob/master/KUBEFLOW_BRAND_GUIDELINES.pdf).
+**Entitlement**
```
[the remark this answers is not in the document]
Comment (james-jwu) [reply proposals/kubeflow-conformance-program-proposal.md:28]: Fixed in #528

File: proposals/kubeflow-conformance-program-proposal.md [hunk 4]
```diff
@@ -17,15 +17,22 @@ This document aims to define conformance criteria for the following usage of Kub
...
+
+- Conformant distributions are entitled to refer to the distribution as "**Certified Kubeflow**". Example usage:
+   - Appending “(Certified Kubeflow)” to the distribution name: e.g. AI-IS-FUN (Certified Kubeflow)
+   - Reference the Certified Kubeflow designation in discussion with customers, or on public documentation
+- Display a logo (to be designed) on the public website and documentation of the distribution
+- Be listed under a partner web page under the Kubeflow project.
+- The naming of the distribution still needs to follow [Kubeflow Brand Guidelines](https://github.com/kubeflow/community/blob/master/KUBEFLOW_BRAND_GUIDELINES.pdf).
 
 The following are out of scope of the conformance tests:
 1. Product quality and supportability
```
[the remark this answers is not in the document]
Comment (james-jwu) [reply proposals/kubeflow-conformance-program-proposal.md:38]: Made a new sub-heading for "Out of scope" in #528

File: proposals/kubeflow-conformance-program-proposal.md [hunk 5]
```diff
@@ -60,7 +67,7 @@ The first version of conformance aims to be inclusive of current components in K
 
 #
```
[the remark this answers is not in the document]
Comment (james-jwu) [reply proposals/kubeflow-conformance-program-proposal.md:68]: Fixed in #528

File: proposals/kubeflow-conformance-program-proposal.md [hunk 6]
```diff
@@ -74,9 +81,15 @@ We expect this test to evolve, due to the ambiguity of “Kubenetes Application
...
 
-The first version of the test verifies that the application is integrated with Kubeflow Pipelines. Pipeline and Metadata and the binding “glue” for other Kubeflow components. Metadata generation is automatic when Kubeflow Application conforms to standard Kubeflow Pipelines component interface.
+The first version of the test verifies that the application is integrated with Kubeflow Pipelines. Pipeline and Metadata are the binding “glue” for other Kubeflow components. Metadata generation is automatic when Kubeflow Application conforms to standard Kubeflow Pipelines component interface.
 
-Kubeflow Application is entitled to refer to the application as "Kubeflow Native" or <TBD>. The application may be listed under an application catalog (to be created) under Kubeflow project. The naming of the application still needs to follow [Kubeflow Brand Guidelines](https://github.com/kubeflow/community/blob/master/KUBEFLOW_BRAND_GUIDELINES.pdf).
+**Entitlement**
+- Conformant applications are entitled to refer to the application as “Kubeflow Native” or <TBD>. Example usage:
+   - Appending “(Kubeflow Native)” to the distribution name: e.g. SUPER-TRAINER (Kubeflow Native)
```
[the remark this answers is not in the document]
Comment (james-jwu) [reply proposals/kubeflow-conformance-program-proposal.md:88]: Good catch. Fixed in #528

File: proposals/kubeflow-conformance-program-proposal.md [hunk 7]
```diff
@@ -74,9 +81,15 @@ We expect this test to evolve, due to the ambiguity of “Kubenetes Application
 
 2. The application-under-test integrates well with Kubeflow.
 
-The first version of the test verifies that the application is integrated with Kubeflow Pipelines. Pipeline and Metadata and the binding “glue” for other Kubeflow components. Metadata generation is automatic when Kubeflow Application conforms to standard Kubeflow Pipelines component interface.
+The first version of the test verifies that the application is integrated with Kubeflow Pipelines. Pipeline and Metadata are the binding “glue” for other Kubeflow components. Metadata generation is automatic when Kubeflow Application conforms to standard Kubeflow Pipelines component interface.
 
-Kubeflow Application is entitled to refer to the application as "Kubeflow Native" or <TBD>. The application may be listed under an application catalog (to be created) under Kubeflow project. The naming of the application still needs to follow [Kubeflow Brand Guidelines](https://github.com/kubeflow/community/blob/master/KUBEFLOW_BRAND_GUIDELINES.pdf).
+**Entitlement**
+- Conformant applications are entitled to refer to the application as “Kubeflow Native” or <TBD>. Example usage:
```
[the remark this answers is not in the document]
Comment (james-jwu) [reply proposals/kubeflow-conformance-program-proposal.md:87]: Fixed in #528

**Comment by james-jwu** [comment]
@ca-scribner @thesuperzapper I propose renaming "Kubeflow Native/Application" to "Kubeflow Pipeline Application", using the currently proposed tests in the RFC. The concerns and questions around "KF Native" can be addressed by changing the naming and scoping to Pipelines. This allows us to move forward with setting up the program, gauge interest from 3rd parties and industry, and can perhaps inform us when we want to expand the scope of the program to KF Native. I will prepare a PR shortly.

**Comment by thesuperzapper** [comment]
@james-jwu I am not sure the current requirements are well suited to a "Kubeflow Pipelines Certified Application", I'd say we should just get rid of the "Kubeflow Native" one and start again for the "Kubeflow Pipelines Certified Application".

Probably the main requirement should be that the application has a well supported pipeline component, which meets some quality standards. This will allow managed offerings like Redshift/BigQuery to be "certified" which would be great for users.

**Pull request merged**